recipes/tb3py/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "tb3py" %}
+{% set version = "2021.7.11" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tb3py-{{ version }}.tar.gz
+  sha256: 6105173883cd6cbf8544413aedf5e5aa4f47bcaa62be9a8ec8c94a951345f9cd
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - requests
+  run:
+    - jarvis-tools
+    - julia
+    - numpy
+    - pytest
+    - python >=3.6
+    - requests
```

Comment:
Any reason not to match the version numbers in the [upstream repo](https://github.com/usnistgov/tb3py/blob/9b0f230146b919c27a9247fdf530ba6a3a4d0e25/setup.py#L54-L58)?